content/en/docs/tasks/debug/debug-application/debug-running-pod.md

```diff
@@ -718,3 +718,59 @@ Clean up the Pod when you're finished with it:
 ```shell
 kubectl delete pod myapp
 ```
+
+### Custom Profile {#custom-profile}
+
+{{< feature-state for_k8s_version="v1.31" state="beta" >}}
+
+You can define a partial container spec as a custom profile in either YAML or JSON format.
+The custom profile can be applied to an ephemeral container or a debugging container in a copied Pod 
+or a debugging Pod using the `--custom` flag.
+
+{{< note >}}
+Custom profile only supports the modification of the ephemeral container or debugging container spec.
+It does not support the modification of the Pod spec.
+Modifications for `command`, `image`, `lifecycle`, `name` and `volumeDevices` fields 
+via a custom profile are not allowed.
+{{< /note >}}
+
+Create a Pod named myapp as an example:
+
+```shell
+kubectl run myapp --image=busybox:1.28 --restart=Never -- sleep 1d
+```
+
+Create a custom profile in YAML or JSON format.
+Here, create a YAML format file named `custom-profile.yaml`:
+
+{{% code_sample file="debug/custom-profile.yaml" %}}
```

Comment:
Please consider embed this YAML directly because it is not a self-contained k8s resource manifest for users to try out.

Reply:
@tengqm 
OK, I've addressed.